libcaf_net/caf/net/defaults.hpp

```diff
@@ -36,4 +36,7 @@ CAF_NET_EXPORT extern const size_t max_header_buffers;
 /// Port to listen on for tcp.
 CAF_NET_EXPORT extern const uint16_t tcp_port;
 
+/// Port to listen on for udp.
+CAF_NET_EXPORT extern const uint16_t udp_port;
```

Comment:
Do we need two constants? I assume this is for exposing BASP. Since we're going to use the locator (URI) as identifier for the CAF node, we must use either TCP or UDP. In other words, one default port would suffice.

Reply:
I thought that BASP could be used with `udp` or `tcp` at the same time, thus both transports require different ports to be accessible from the network.

Reply:
In the new model, the node ID *is* the locator in order to get rid of the pseudo-DNS CAF is currently doing. This means there must be exactly one valid way to connect to a BASP instance.

Reply:
I forgot about the fixed node ID.. Still I am not sure if that is the best way to solve this problem. Is it really necessary to allow only one possible way to reach a node via `BASP`?

Reply:
By definition, a node can only have a single ID. If we allow multiple IDs pointing to the same BASP, we have the exact same issues we wanted to get rid of with the redesign that @josephnoir initiated.

We could of course offer a "connect there and see who's responding" function, but it would convolute the design quite a bit. Because then, we can't use the ID any longer to identify connections. I honestly don't see an upside here that would be worth the troubles in the first place.

Reply:
I can understand that. I'll remove the duplicate port field when I'm fixing the other remark.